tests/gbs/test_plot.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Unit tests for strawberryfields.gbs.plot
+"""
+# pylint: disable=no-self-use,unused-argument,too-many-arguments
+import itertools
+from collections import Counter
+
+import networkx as nx
+import numpy as np
+import pytest
+
+from strawberryfields.gbs import plot
+
+
+class TestNodeCoords:
+    """Test for the internal function strawberryfields.gbs.plot._node_coords"""
+
+    @pytest.mark.parametrize("dim", [4, 5, 6])
+    def test_node_coords(self, dim):
```

Comment:
I enjoyed this test 😁

This is nitpicking, but if we only need one test for this function, then we could probably do away with the `TestNodeCoords` class and just have the inner function.

^ This is not important though

Reply:
Done!